### DISCUSSION TITLE
Python Code Sample 2

### DISCUSSION THREAD

**Discussion by pdp2121** [XRPLBounties/Proposals#79] [Open Bounties]
### Discussion thread for bounty: [Python Code Sample 2](https://github.com/XRPLBounties/Proposals/pull/78)
Add more code samples, and fix depreciated ones to capture more common tasks from users.

**Comment by ObiajuluM** [comment]
hello @pdp2121 I already completed the python escrow history bounty

https://github.com/XRPLF/xrpl-dev-portal/blob/master/content/_code-samples/escrow/py/account_escrows.py

filtered by source and destination.

how do i go about it?

**Comment by jonathanlei** [comment]
[This bounty](https://github.com/XRPLBounties/Proposals/blob/main/open_bounties/0079%20Python%20Code%20Samples%202.md) is now approved and feel free to submit the work you have! Thanks

**Comment by JST5000** [comment]
We've accepted @wojake's contributions for milestones 1, 4, 5, 6 and 7:
* https://github.com/XRPLF/xrpl-dev-portal/pull/1682
* https://github.com/XRPLF/xrpl-dev-portal/pull/1683
* https://github.com/XRPLF/xrpl-dev-portal/pull/1684
* https://github.com/XRPLF/xrpl-dev-portal/pull/1682
* https://github.com/XRPLF/xrpl-dev-portal/pull/1685

Please claim those bounties by filling out this form if you haven't already :) - [https://xrpl.typeform.com/xrpl-bounties](https://xrpl.typeform.com/xrpl-bounties?utm_source=xxxxx&utm_medium=xxxxx&utm_campaign=xxxxx&utm_term=xxxxx&utm_content=xxxxx)

**Reply by JST5000** [reply]
(Edited to include 6 since I missed that on my first comment, sorry about that!)

**Comment by rikublock** [comment]
Interested in doing milestone 3 (Decentralized Exchange), but not quite sure what's the task exactly.
The second link in the description [here](https://github.com/XRPLBounties/Proposals/blob/main/open_bounties/0079%20Python%20Code%20Samples%202.md) might be wrong?

**Reply by pdp2121** [reply]
Hi @rikublock, thanks for your interest and sorry about the link. I will update it now.

**Comment by pdp2121** [comment]
We have accepted @rikublock submission for milestone 2:
https://github.com/XRPLF/xrpl-dev-portal/pull/1748

**Reply by rikublock** [reply]
Thanks. 9237

**Comment by JST5000** [comment]
We have accepted @rikublock's submission for milestone 3:
https://github.com/XRPLF/xrpl-dev-portal/pull/1751

(Which is the last milestone - this bounty is now completed!)
